sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java

```diff
@@ -141,6 +143,34 @@ public CaseDetailedTableViewSteps(
               "Reporting user is not correct");
           softly.assertAll();
         });
+
+    When(
+        "I back to Case Directory using case list button",
+        () -> webDriverHelpers.clickOnWebElementBySelector(BACK_TO_CASES_LIST_BUTTON));
+
+    When(
+        "I check if Case date format is correct",
+        () -> {
+          List<Map<String, String>> tableRowsData = getTableRowsData();
+          Map<String, String> detailedCaseDTableRow = tableRowsData.get(0);
+          softly.assertTrue(
```

Comment:
add a more specific failure message for both asserts, to understand from results which date is wrong